tools/topology/sof-tgl-max98373-rt5682.m4

```diff
@@ -94,10 +94,21 @@ dnl     frames, deadline, priority, core)
 
 # Low Latency playback pipeline 2 on PCM 1 using max 2 channels of s32le.
 # Schedule 48 frames per 1000us deadline on core 0 with priority 0
-PIPELINE_PCM_ADD(sof/pipe-volume-playback.m4,
+ifelse(XPERI, `1',
+`
+PIPELINE_PCM_ADD(sof/pipe-eq-iir-volume-playback.m4,
         2, 1, 2, s32le,
 	1000, 0, 0,
 	48000, 48000, 48000)
+'
+,
+`
```

Comment:
Needs a comment for each item you are adding here.

Reply:
Added comment for pipeline that was missing